src/components/topicPage/Cards.js

```diff
@@ -0,0 +1,178 @@
+'use strict'
```

Comment:
try to modify `src/components/topic/Card.js` and `src/components/topic/Cards.js`, rather than to create a new one.